src/components/modules/toolbar/conversion.ts

```diff
@@ -276,64 +278,98 @@ export default class ConversionToolbar extends Module<ConversionToolbarNodes> {
     Array
       .from(tools.entries())
       .forEach(([name, tool]) => {
-        const toolboxSettings = tool.toolbox;
         const conversionConfig = tool.conversionConfig;
 
-        /**
-         * Skip tools that don't pass 'toolbox' property
-         */
-        if (_.isEmpty(toolboxSettings) || !toolboxSettings.icon) {
-          return;
-        }
-
         /**
          * Skip tools without «import» rule specified
          */
         if (!conversionConfig || !conversionConfig.import) {
           return;
         }
 
-        this.addTool(name, toolboxSettings.icon, toolboxSettings.title);
+        if (Array.isArray(tool.toolbox)) {
```

Comment:
Maybe we can unify that to array somewhere at the start, so we don't need to add checks everywhere